mappings/net/minecraft/world/biome/PlainBiome.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_100 net/minecraft/world/biome/PlainBiome
```

Comment:
I suggest a name change to "BasicBiome" to avoid confusion with the plains biome. This name is probably not perfect but it is hard to name this class accurately due to it being used for multiple different biomes.

Reply:
Ye i think this would be a more fitting name too

Reply:
Renamed to BasicBiome

Reply:
I'm fine with this, as there's no direct yarn counterpart.